_delphi_utils_python/delphi_utils/validator/dynamic.py

```diff
@@ -254,6 +256,32 @@ def check_max_allowed_max_date(self, max_date, geo_type, signal_type, report):
 
         report.increment_total_checks()
 
+    def check_reference_api_df(self, reference_api_df, geo_sig_df, reference_end_date):
```

Comment:
Looks like this isn't actually checking anything, so a different name would be more suitable, e.g. "pad_to_date" or something.